### DISCUSSION TITLE
NuGet : Easy include advanced astrological calculations into your projects

### DISCUSSION THREAD

**Discussion by sengiv** [gen-so/Genso.Astrology#18] [Announcements]
![image](https://user-images.githubusercontent.com/43817262/226801801-eba6fa42-d0b5-42e6-8ee6-281da16aaa19.png)

Now it's easy for developers to include **advanced astrological calculations** into their own projects.

## Get Started
Install via simple command
```powershell
dotnet add package VedAstro.Library
```

**Comment by gokul6350** [comment]
Make one library for python that would be help full

**Reply by sengiv** [reply]
Brilliant idea 💡 👏 
The code is currently all in C#, do you have any suggestions on how we can make python lib from it?

**Reply by gokul6350** [reply]
Actually, I don't know much about C# but we can use https://github.com/pythonnet/pythonnet for this purpose

**Reply by sengiv** [reply]
check out the dev progress for this feature #34

**Comment by HemKal** [comment]
I guess @sengiv  already provided that as part of REST API. Isn't it ? And secondly his calculations are based on different Aynamsa

**Reply by sengiv** [reply]
Yes, API should have all calls from Library.
But this is still under development.
And only 10% has been exposed via API atm. Hopefully with support from users like you @HemKal 
We can aim to close the gap, to 80% by year end.

**Reply by HemKal** [reply]
WIll try to jump in @sengiv .. Keep up the good work.

**Reply by sengiv** [reply]
join our slack bro --> can discuss easy and you get better idea what is going on internally

https://join.slack.com/t/vedastro/shared_invite/zt-1u7pdqjky-hrJZ7e3_vM2dZOmVY8FeHA

**Reply by sengiv** [reply]
@HemKal regarding the **Ayanamsa**....after many request, we going to enable  "**Custom Ayanamsa**".
This allow users to input others like, "Lahiri" and also make their own! You can even make your own Ayanamsa and use it in the whole system. From API to Chart.
Track task progress here --> #79

**Reply by sengiv** [reply]
Custom Ayanamsa
**https://vedastro.org/Account/**
![image](https://github.com/VedAstro/VedAstro/assets/43817262/638bff02-62b8-40ec-b262-5aa97ff56819)
